commec/regulation/containers.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+
+"""
+Defines Containers to show default layouts for regulation data.
+What is the shape of the regulation data, and where to store it.
+
+    RegulationContainer : 
+        A standard output that may be associated with a hit.
+        Contains list, region, and level information.
+    RegulationLevel :
+        Determines valid level information strings.
+    TaxidRegulationContainer :
+        Storage of all regulation data for a given taxid from a given list.
+
+Also defines the storage data for 
+    * regulation lists, 
+    * per taxid regulation information per list, 
+    * and child to parent taxid mapping.
+"""
+import logging
+import re
+from dataclasses import dataclass, field, fields
+from typing import Optional
+from enum import StrEnum
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+class AccessionFormat(StrEnum):
+    """
+    Supported hashable accesion formats.
+    """
+    TAXID = "TaxonomyID"
+    GENBANK = "GenbankProtein"
+    UNIPROT = "UniprotID"
```

Comment:
Let's remove these since we don't actually support them yet, and I don't want anyone (including me in 3 months) to be misled